Dockerfile

```diff
@@ -3,5 +3,10 @@ FROM busybox:ubuntu-14.04
 RUN mkdir -p /opt
 ADD ./aws-operator /opt/aws-operator
 
+RUN mkdir -p /opt/resources
```

Comment:
But this means I would manually need to put the template into `/opt/...` right? So whenever I change the file I need to remember to move the file again and my brain will definitely not remember this.